src/azure-cli/azure/cli/command_modules/appconfig/_format.py

```diff
@@ -61,6 +65,16 @@ def _configstore_format_group(item):
     ])
```

Comment:
I think we should update this format to show some of the new soft-delete properties too.